### ISSUE TITLE
Accordion labels should outline on hover

### ISSUE THREAD

**Issue by hokei** [NYPL/nypl-design-system#471]
### Description of issue:
- The labels in Accordion do not outline on hover, so a keyboard user does not have visual feedback on which element is selected.

### Component in question (if applicable):
- Accordion

**Comment by helenvholmes** [comment]
Hey Crystal, can you add some clarification? Is the issue on the `:hover` event or the `:active` event? From the note on keyboard users it sounds like we need something for the `:active` event, which admittedly that element might not have natively (although we can get it added).

**Comment by hokei** [comment]
Here's a video of the behavior:  
As you see, when you tab over the link, there's some visual indication that it's active.  However, when you tab over the label, there is no such indication until you open/close it. 

https://user-images.githubusercontent.com/3579472/102826698-2f7d2300-43af-11eb-9e7e-dcd8f56792cb.mov

**Comment by helenvholmes** [comment]
Closing as dup of https://jira.nypl.org/browse/UX-240
